src/mocks/data.ts

```diff
@@ -4792,3 +4792,60 @@ export const mockFragment = (pageId: string) => ({
     ],
     published: true
 });
+
+export const mockPage = (pageId: string) => ({
+    id: pageId,
+    type: "page",
+    canonicalSourceFile: `content/mocks/${pageId}.json`,
+    encoding: "markdown",
+    title: `Mock page: ${pageId}`,
+    children: [
+      {
+        type: "content",
+        value: "Here is some mock content for you!",
+        encoding: "markdown",
+        published: true
+      }
+    ],
+    published: true
+  });
+
+  export const mockAssignedQuiz = [{
+    id: 1,
+    quizId: "example_quiz_1",
+    quizSummary: {
+        id: "example_quiz_1",
+        title: "Example Quiz 1",
+        type: "isaacQuiz",
+        level: 2,
+        tags: [],
+        url: "/isaac-api/api/quiz/example_quiz_1",
+        visibleToStudents: false},
+    groupId: 1,
+    ownerUserId: mockUser.id,
+    creationDate: 888888888888 as unknown as Date,
```

Comment:
Is this type casting actually needed?

Reply:
Several MSW warnings like below were being triggered when running the tests:

`    console.warn
      [MSW] Warning: captured a request without a matching request handler:
      
        • GET http://localhost:8080/isaac-api/api/quiz/assigned`
        
So new handlers were added, and the data was there to mock what would come back:

`    rest.get(API_PATH + "/quiz/assigned", (req, res, ctx) => {
        return res(
            ctx.status(200),
            ctx.json(mockAssignedQuiz)
        );
    }),`

Reply:
I mimicked the structure of other mock data, and without the "unknown" part it gives error "Conversion of type 'number' to type 'Date' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first."